apps/music/views/songs/view.js

```diff
@@ -8,47 +8,65 @@ var SongsView = View.extend(function SongsView() {
   this.searchResults = document.getElementById('search-results');
   this.list = document.getElementById('list');
 
-  this.searchBox.addEventListener('search', (evt) => this.search(evt.detail));
+  this.client.on('databaseChange', () => this.update());
+
+  this.configureList();
+  this.update();
+});
 
-  this.searchResults.addEventListener('open', () => {
+SongsView.prototype.configureSearch = function() {
+  var results = this.searchResults;
+  var box = this.searchBox;
```

Comment:
See same comments as above in "apps/music/views/artists/view.js". Now that we have fat-arrow functions, we shouldn't need to declare vars all over the place like this to maintain scope. Adding vars like this can make the code more difficult to follow since it might not be obvious what `results` is referring to in later code.